src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -6,7 +6,8 @@
         <h2>Company Name</h2>
       </header>
 
-      <div>** Correct Name component goes here **</div>
+      <div><name-request-info /></div>
+      <div><name-translations /></div>
```

Comment:
Our style is to use the PascalCase version of the component names here, so that VS Code highlights them nicely (and sometimes you can click on them to go to their definition).

Also, don't add divs around them if not needed.

Reply:
Thank you! Fixed

Reply:
@ArwenQin , don't resolve conversations that you didn't open. I use open conversations as something I need to re-review, so that I don't have to review everything again. Let me close the conversations I opened.

Reply:
This conversation is resolved.